packages/plugin-eslint-rules/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "@grafana/plugin-eslint-rules",
+  "version": "0.0.4",
+  "description": "An ESlint plugin with Grafana plugin specific rules",
+  "main": "./dist/index.js",
+  "files": [
+    "dist/"
+  ],
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org/",
+    "access": "public"
+  },
+  "type": "commonjs",
+  "exports": {
+    ".": {
+      "default": "./dist/index.js"
+    },
+    "./package.json": "./package.json"
+  },
+  "keywords": [
+    "eslint",
+    "eslintplugin",
+    "grafana"
+  ],
+  "homepage": "https://www.npmjs.com/package/@grafana/plugin-eslint-rules",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/grafana/plugin-tools"
+  },
+  "bugs": "https://github.com/grafana/plugin-tools/issues",
```

Comment:
We can also update the [bug_report template](https://github.com/grafana/plugin-tools/blob/main/.github/ISSUE_TEMPLATE/bug_report.yml?plain=1#L14) to include this package as an option.

Reply:
Fixed 👍